tests/pipelines/test_pipelines_common.py

```diff
@@ -1351,14 +1351,30 @@ def _test_attention_slicing_forward_pass(
 
         pipe.enable_attention_slicing(slice_size=1)
         inputs = self.get_dummy_inputs(generator_device)
-        output_with_slicing = pipe(**inputs)[0]
+        output_with_slicing1 = pipe(**inputs)[0]
+
+        pipe.enable_attention_slicing(slice_size=2)
+        inputs = self.get_dummy_inputs(generator_device)
+        output_with_slicing2 = pipe(**inputs)[0]
+
+        pipe.enable_attention_slicing(slice_size=3)
```

Comment:
can we remove the `slice_size=3` test? I think the CI would pass without this, no?